content/manager_webui/default-widgets-ref.md

```diff
@@ -59,16 +59,21 @@ The blueprint ID must be passed to the widget. This can be done either in the pa
 Displays all the local blueprints on the Cloudify Manager in the context of the current tenant as a table or tile view, according to the logged-in user's permissions. In the catalog view, each blueprint entry includes the icon PNG file with which it was uploaded or the Cloudify default icon if no PNG file was attached.
 
 #### Configuration
-In the widget configuration dialog, you can toggle on the **Click to drill down** option so that, when the blueprint is clicked, the drill down page is displayed for that blueprint.
+In the widget configuration dialog, you can toggle on the **Enable click to drill down** option so that, when the blueprint is clicked, the drill down page is displayed for that blueprint.
 
 ![blueprints-list]({{< img "ui/widgets/blueprints-list.png" >}})
 
 ### Blueprints Catalog
 
-Displays the details of a blueprint that exists in a repository under a Github user that has been specified in the widget's settings, as a table or tile view. The widget provides an `upload` option for each of the blueprints, enabling them to be easily uploaded locally to the Manager.
+Displays blueprints (as a table or tile view) that exists in a repository under a Github account. 
+
+#### Configuration
+Name of that account (user of organization) has to be specified in the widget's settings. The widget provides an `upload` option for each of the blueprints, enabling them to be easily uploaded locally to the Manager. 
 
 You can create a filter query in the configuration to specify the blueprints that appear. 
 
+You can also provide Github username (and password) which will be used for data fetching. These parameters are fetched from secrets (see [Secrets Store Management]({{< relref "manager_webui/default-widgets-ref.md#secrets-store-management" >}})) as `github-username` and `github-password` keys. Providing those secrets is necessary eg. if you want to access private repositories. 
```

Comment:
You can also enter a Github credentials to let you fetch data. These parameters are pulled from [secrets]({{< relref "manager_webui/default-widgets-ref.md#secrets-store-management" >}})) as the `github-username` and `github-password` keys. You must enter those secrets to access private repositories.

Reply:
Updated.